lib/SILOptimizer/Transforms/CSE.cpp

```diff
@@ -731,6 +731,10 @@ bool CSE::processLazyPropertyGetters() {
     assert(getter && getter->isLazyPropertyGetter());
     SILBasicBlock *callBlock = ai->getParent();
 
+    // We cannot inline a non-ossa function into an ossa function
+    if (ai->getFunction()->hasOwnership() && !getter->hasOwnership())
+      continue;
```

Comment:
It would be better to make the check in `static bool isLazyPropertyGetter(ApplyInst *ai)`. All other checks are also done in this function.

Reply:
Done. Thanks